README.md

```diff
@@ -39,8 +39,8 @@ clean postfix installation.
 
 **WARNING**: If you specify `previous: replaced`, the role reinstalls the postfix
 package and replaces the existing `/etc/postfix/main.cf` and
-`/etc/postix/master.cf` files. Ensure to back up those files to preserve your
-settings.
+`/etc/postix/master.cf` files. <!--- wokeignore:rule=master -->
```

Comment:
There is a typo here (was here previously) - should be `/etc/postfix/master.cf` - please fix that too